helm/examples/efs/README.md

```diff
@@ -1,51 +1,69 @@
 # HPCC Elastic File System (EFS) Storage
 
-## EFS Server Settings
-Make sure an EFS server is available or create one before using this chart.<br/>
-EFS Server settings should be set in the file efs-env<br/>
-The chart requires <EFS region> and <EFS ID>
-The <EFS ID> can be found in AWS Console EFS service or from AWS Cli:
-```console
-  aws efs describe-file-systems --region <EFS region> | grep "^FILESYSTEMS" |  awk -F $'\t' '{print $8, $5}'
-```
-The output will display EFS name and ID.
 
-It is recommended to provide the option "--managed" and a security group id during EKS cluster creation. EFS Server can use the same security group or if you know your security group of node pools they are OK to use for EFS security group. Otherwise you need to provide the following:
-- EKS cluster name
-```console
-kubectl config get-clusters | cut -d'.' -f1
+
+## EFS CSI Driver
+"examples/efs" requires Elastic FileSystem (EFS) Container Storage Interface (CSI) driver.
+We provide some scripts to simplified the process. For detail please referencing https://docs.aws.amazon.com/eks/latest/userguide/efs-csi.html
+Current implementation only tested successfully with a IAM user. It may need update for Active Directory Federation Service (ADFS)
+
+### Prepare environment
+Make sure an EFS server is available or create an [EFS server](#efs-server) before using this chart.<br/>
+Fill following information in efs-env
+```code
+ACCOUNT_ID       # AWS Account ID
+EKS_NAME         # Elastic Kubernetes Service (EKS) cluster name: kubectl configure get-cluster  (The first part of the name before ".")
+EFS_ID           # EFS ID can be found from AWS Console or command-line:
+                   aws efs describe-file-systems --query "FileSystems[*].FileSystemId" --output text
+EFS_REGION       # EFS region. We strongly suggest you have the same region for EFS server and EKS cluster
+EFS_SECURITY_GROUPS  # EFS seucurity group which is the subnet security group which should be same as EKS
+EFS_BASE_PATH    # Base directory to mount in the EFS. If you use EFS for multiple HPCC clusters you should set this differently to avoid overwriting each others
```

Comment:
to avoid overwriting each others -> to avoid clashing with the other clusters

Reply:
fixed